source/config.js

```diff
@@ -235,6 +239,25 @@ if (typeof currentRootPath !== 'string') {
 }
 module.exports.rootPath = currentRootPath;
 
+let currentGitCommand = module.exports.gitCommand;
+if (typeof currentGitCommand !== 'string') {
+  currentGitCommand = '';
+} else if (currentGitCommand !== '') {
+  // must start with a slash
+  if (currentGitCommand.charAt(0) !== '/') {
+    currentGitCommand = '/' + currentGitCommand;
+  }
+  // can not end with a trailing slash - it should be a command
+  if (currentGitCommand.charAt(currentGitCommand.length - 1) === '/') {
+    currentGitCommand = currentGitCommand.substring(0, currentGitCommand.length - 1);
+  }
+}
+// finally set default if empty
+if (currentGitCommand === '') {
```

Comment:
Why? If set to `null` or empty string it will get right there - maybe I'm missing something, but this is the case I've added that final check. It is actually outside of that `if` (on lines 243..254).